### PR TITLE
kosaraju + scc upd

### PR DESCRIPTION
**Pull request by nwatx**

### CODE REVIEW

File: content/6_Advanced/SCC.mdx [hunk 1]
```diff
@@ -43,78 +41,71 @@ frequency: 1
...
 
-Just run
-[Kosaraju's](<https://github.com/bqi343/USACO/blob/master/Implementations/content/graphs%20(12)/DFS/SCCK.h>)
-or
-[Tarjan's](<https://github.com/bqi343/USACO/blob/master/Implementations/content/graphs%20(12)/DFS/SCCT.h>)
-SCC algorithm on the graph.
-
-Then assign each component an $ID$ (starting from $1$).
+Notice the definition of a strongly connected component is "if every vertex is reachable from every other vertex," which happens to be the problem statement.
+Therefore, simply run [Kosaraju's Algorithm](<https://github.com/bqi343/USACO/blob/master/Implementations/content/graphs%20(12)/DFS/SCCK.h>) or [Tarjan's Algorithm](<https://github.com/bqi343/USACO/blob/master/Implementations/content/graphs%20(12)/DFS/SCCT.h>)
```
Comment (bqi343) [inline content/6_Advanced/SCC.mdx:45]: these are linked above so no need to link them again?

File: content/6_Advanced/SCC.mdx [hunk 2]
```diff
@@ -43,78 +41,71 @@ frequency: 1
...
-		for(int x : todo) if (comp[x] == -1) {
-			dfs2(x,x), comps.push_back(x);
-		}
+
+using vi = vector<int>;
+#define pb push_back
+
+const int mx = 1e5+1;
+
+vi adj[mx], adj_t[mx], S;
```
Comment (bqi343) [inline content/6_Advanced/SCC.mdx:63]: what does `adj_t` stand for

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: content/6_Advanced/SCC.mdx (hunk 2 above)
Comment (nwatx) [reply content/6_Advanced/SCC.mdx:63]: adj (transpose). I'll make a note

File: content/6_Advanced/SCC.mdx (hunk 1 above)
Comment (nwatx) [inline content/6_Advanced/SCC.mdx:45]:
```suggestion
Therefore, simply run Kosaraju's Algorithm or Tarjan's Algorithm
```

**Review by bqi343 — APPROVED** [review]

**Pull request merged**